src/test/java/gov/healthit/chpl/aqa/stepDefinitions/LoginSteps.java

```diff
@@ -79,6 +79,9 @@ public void logInAsRole(final String role) {
         LoginPage.password(getDriver()).sendKeys(password);
         LoginPage.loginButton(getDriver()).click();
         getWait().until(ExpectedConditions.visibilityOf(LoginPage.logoutButton(getDriver())));
```

Comment:
Why are there two different "wait"s here? One from the Base.java page, found in the "getWait()" call, and the other created in this method, on line 72. There should only be one, and it should probably be the one from the getWait method

Reply:
I think it's been there from the time before we had Base.java, but agree that getWait() should be used now that we have it and is used in other tests